docs/ipr/scripting/RNA_Expression_Similarity.ipynb

```diff
@@ -0,0 +1,4625 @@
+              "      <td>-9.9658</td>\n",
+              "      <td>-9.9658</td>\n",
+              "      <td>-9.9658</td>\n",
+              "      <td>-9.9658</td>\n",
+              "      <td>-9.9658</td>\n",
+              "      <td>-9.9658</td>\n",
+              "      <td>-9.9658</td>\n",
+              "      <td>-9.9658</td>\n",
+              "      <td>-9.9658</td>\n",
+              "      <td>-9.9658</td>\n",
+              "      <td>-9.9658</td>\n",
+              "      <td>-9.9658</td>\n",
+              "      <td>-9.9658</td>\n",
+              "      <td>-9.9658</td>\n",
+              "      <td>-9.9658</td>\n",
+              "    </tr>\n",
+              "  </tbody>\n",
+              "</table>\n",
+              "<p>60498 rows × 936 columns</p>\n",
+              "</div>"
+            ],
+            "text/plain": [
+              "                    TCGA-G3-A3CH-11  ...  TCGA-FV-A3I0-11\n",
+              "gene                                 ...                 \n",
+              "ENSG00000242268.2           -9.9658  ...          -9.9658\n",
+              "ENSG00000259041.1           -9.9658  ...          -9.9658\n",
+              "ENSG00000270112.3           -9.9658  ...          -9.9658\n",
+              "ENSG00000167578.16           3.5572  ...           3.0876\n",
+              "ENSG00000278814.1           -9.9658  ...          -9.9658\n",
+              "...                             ...  ...              ...\n",
+              "ENSG00000273233.1           -9.9658  ...          -9.9658\n",
+              "ENSG00000105063.18           3.4358  ...           3.2174\n",
+              "ENSG00000231119.2           -2.5479  ...          -3.0469\n",
+              "ENSG00000280861.1           -9.9658  ...          -9.9658\n",
+              "ENSG00000181518.3           -9.9658  ...          -9.9658\n",
+              "\n",
+              "[60498 rows x 936 columns]"
+            ]
+          },
+          "metadata": {
+            "tags": []
+          },
+          "execution_count": 8
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "HBCmQl9OTLVM"
+      },
+      "source": [
+        "Now that we have our matrix we are ready to start the correlation"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "i6Ptf4p9TXs6"
+      },
+      "source": [
+        "## Spearman Correlation\n",
+        "\n",
+        "The input for an analysis where a patient sample is being compared to the TCGA cohort of interest would typically be an expression matrix using the same normalization method as the comparator cohort. Note: Differences in library preparation, sequencing and bioinformatics pipelines can lead to variation in expression quantification. It is important to understand the impact these differences have on expression quantification before comparing expression values that are produced using different pipelines.\n",
```

Comment:
maybe change 'Note: Differences' to ' Note that differences'. Because 'Note' is used elsewhere in the docs as a tag/section name it's weird to see it in a sentence here